packages/core/react-loosely-lazy/src/lazy/placeholders/hydrate.tsx

```diff
@@ -19,15 +19,14 @@ export const PlaceholderFallbackHydrate = ({
           key: String(i),
         });
         // text node
-        if (!tagName)
-          return React.createElement(Fragment, props, el.textContent);
+        if (!tagName) return createElement(Fragment, props, el.textContent);
```

Comment:
/nit

I would prefer breaking a line of adding `{` `}` for readability.

Reply:
I do prefer that style as well, but this is how it was originally written. It should be enforced with the curly eslint rule, but I also like minimising custom config and it's not part of the recommended preset.